programs/monaco_protocol/src/state/market_position_account.rs

```diff
@@ -12,6 +12,20 @@ pub struct MarketPosition {
     pub market_outcome_sums: Vec<i128>,
     pub outcome_max_exposure: Vec<u64>,
     pub payer: Pubkey, // solana account fee payer
+    pub matched_risk: u64,
+    pub matched_risk_per_product: Vec<ProductMatchedRiskAndRate>,
```

Comment:
Should these be available in the client?

Reply:
I do not think so. Only modifiable by Monaco itself.